pkg/console/operator/sync_v400.go

```diff
@@ -335,22 +335,22 @@ func secretAndOauthMatch(secret *corev1.Secret, client *oauthv1.OAuthClient) boo
 // update cluster operator status... i believe this
 // should be automatic so long as the CR.Status is
 // properly filled out with the appropriate values.
-func setStatus(cs v1alpha1.ConsoleStatus, svc *corev1.Service, rt *routev1.Route, cm *corev1.ConfigMap, dep *appsv1.Deployment, oa *oauthv1.OAuthClient, sec *corev1.Secret) {
+func setStatus(cs v1alpha1.ConsoleOperatorConfigStatus, svc *corev1.Service, rt *routev1.Route, cm *corev1.ConfigMap, dep *appsv1.Deployment, oa *oauthv1.OAuthClient, sec *corev1.Secret) {
```

Comment:
Status does need to be set, this will be revised in a follow-on when we add back the `ClusterOperator` object to report operator status.